lrpar/src/lib/ctbuilder.rs

```diff
@@ -349,8 +349,8 @@ where
             None => {
                 // The user hasn't specified a module name, so we create one automatically: what we
                 // do is strip off all the filename extensions (note that it's likely that inp ends
-                // `y.rs`, so we potentially have to strip off more than one extension) and then
-                // add `_y` to the end.
+                // wth `y.rs`, so we potentially have to strip off more than one extension) and
```

Comment:
I think this commit just introduces a typo but doesn't do anything else.

Reply:
Just drop this commit and we should be good.

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/144/commits/4a90be7695e54aece624951b52dffdb8f928ee22